docker/Dockerfile.frontend

```diff
@@ -0,0 +1,14 @@
+# syntax=docker/dockerfile:1
+
+FROM node:16.14.0-buster
+ENV NODE_ENV=production
+
+WORKDIR /app
+
+COPY ["frontend/package.json", "frontend/package-lock.json*", "./"]
```

Comment:
Whole app could be copied in one entry, because later you copy it anyways

Reply:
Done